### PR TITLE
add styling (h/t luiza)

### PR DESCRIPTION
**Pull request by daleluiza**
uploaded style.css with some styles but we'll have to update the views to reflect the new elements, i have those as html pages wonder if i should throw here/i emailed!

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.textbox {
+  padding-top: 1em;
+  margin: 0 auto;
+  height: auto;
+  width: 50%;
+}
+
+.a {
```
Comment (willisplummer) [inline css/styles.css]: lets not call these `a` and `b` -- it's too similar to the anchor element's name + confusing imo.. could we call em left and right or something instead?

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  text-align: left;
+}
+
+.b {
+  padding: 1em;
+  display: block;
+  text-align: right;
+}
+
+.acredit {
```
Comment (willisplummer) [inline css/styles.css]: whatever we do for a and b lets be sure to update these calsses too

File: css/styles.css (hunk 1 above)
Comment (daleluiza) [reply css/styles.css]: totally, let's do that. so left and right and then something like leftcredit and rightcredit to indicate the names of the people in the conversation. im proposing that we dont repeat the names with every line but just have them as a key, once on the screen... whatcha think?

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  padding: 1em;
+  position: fixed;
+  bottom: 1em;
+  right: 1em;
+  display: inline;
+  text-align: right;
+}
+
+/* unvisited link */
+a:link {
```
Comment (willisplummer) [inline css/styles.css]: i'm not familiar with the `:link` selector -- what's that mean?

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+body, p {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "helvetica neue", helvetica, arial, sans-serif;
+  font-size: 18px;
+  line-height: 24px;
```
Comment (willisplummer) [inline css/styles.css]: can we make this in em too so that it's relative to the font-size specified above?

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  letter-spacing: .1em;
+  background: white;
+  text-transform: lowercase;
+}
+
+.homebox {
+  padding-top: 2em;
+  padding-left: 2em;
+  width: 50%;
+  /* border: 1px red solid; */
```
Comment (willisplummer) [inline css/styles.css]: we don't want this?

File: css/styles.css (hunk 1 above)
Comment (willisplummer) [reply css/styles.css]: that makes sense to me about the names -- can do whatever if you show me how it needs to be laid out

File: css/styles.css (hunk 5 above)
Comment (daleluiza) [reply css/styles.css]: we dont want the border, i was just using it to see what was up w my spacing and then turned it off, shall delete

Comment (daleluiza) [reply css/styles.css]: is this what you mean?

File: css/styles.css (hunk 3 above)
Comment (daleluiza) [reply css/styles.css]: just means "a", the link before you click it i think... probs the same as active, there's some redundancy there but think its ok

File: css/styles.css (hunk 1 above)
Comment (daleluiza) [reply css/styles.css]: yup! take a look at the html pages when u get a sec

File: css/styles.css [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  color: blue;
+}
+
+/* selected link */
+a:active {
+  color: blue;
+}
+
+@media screen and (max-width: 700px) {
+  .box {
```
Comment (willisplummer) [inline css/styles.css]: what's this class that's only selected on like tablet and below screen sizes?

**Comment by willisplummer** [comment]
hmm looks like the ds_store got committed - gonna delete and add that to the gitignore.

File: css/styles.css (hunk 5 above)
Comment (willisplummer) [reply css/styles.css]: yep!

**Comment by willisplummer** [comment]
Okay i'm gonna go ahead and merge

File: css/styles.css (hunk 6 above)
Comment (daleluiza) [reply css/styles.css]: ah its messed up, should be for .textbox not .box, ill update. but basically this makes it so on desktop the container is a certain size, centered and on mobile it takes up the entire screen

**Pull request merged**